AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -850,7 +854,6 @@ - (void)showConfirmationAlert:(MSReleaseDetails *)details {
 
   // Displaying alert dialog. Running on main thread.
   dispatch_async(dispatch_get_main_queue(), ^{
-
```

Comment:
Revert this change.

Reply:
It'll pop up in another PR so it should be fine to have it here.